src/Components/SocialIconsEmailPlacement.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from 'react-router-dom'
+import '../Styling/SocialIconsEmailPlacement.css';
+
+function SocialIconsEmailPlacement(){
```

Comment:
I suggest changing the name of this file/component. The current name is confusing and misleading